### DISCUSSION TITLE
Multiple geo_timeseries.h5 stitching

### DISCUSSION THREAD

**Discussion by stoormgeo** [insarlab/MintPy#1142] [Q&A]
Hi, 
What the best routine to stitching multiple georeferenced timeseries files with same reference points? Does image_stitch.py support such functionality?

**Comment by leeyhgeo** [comment]
I think this task is not possible at this time. You can check this [https://groups.google.com/g/mintpy/c/iohsaO72s5Q/m/0AEPbGWoAQAJ](https://groups.google.com/g/mintpy/c/iohsaO72s5Q/m/0AEPbGWoAQAJ) for Yunjun's explanation.
